uber/templates/preregistration/preregbase.html

```diff
@@ -106,7 +106,7 @@
                 .slice(index, 1 + index)
                 .addClass('active');
             (type.onClick || $.noop)();
-            if (type.extra !== undefined && type.extra !== null) {
+            if (type.extra !== undefined && type.extra !== null && type.extra > $.field('amount_extra').val()) {
```

Comment:
`c.DONATIONS_ENABLED` does indeed remove the amount_extra dropdown if set to False. I've added a check to prevent that creating an error - plus another in another part of the JS that was causing its own error.